src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js

```diff
@@ -26,6 +26,17 @@ const ImplementationCmsPageEditComponent = function (
         name: 'Public',
     }];
 
+    $ctrl.descriptionPositions = [{
+        value: 'replace',
+        name: 'Overwrite default page content',
+    }, {
+        value: 'before',
+        name: 'Show before default page content',
+    }, {
+        value: 'after',
+        name: 'Show after default page content',
+    }];
```

Comment:
words above need translation

Reply:
I think these sentences would be good translations

'Standaard pagina-inhoud overschrijven' (overwrite)

'Voor standaard pagina-inhoud tonen' (before)

'Na standaard pagina-inhoud tonen' (after)

Reply:
@Dominique2002 thank you!